src/net_processing.cpp

```diff
@@ -256,6 +256,9 @@ class PeerManagerImpl final : public PeerManager
                         const std::chrono::microseconds time_received, const std::atomic<bool>& interruptMsgProc) override;
 
 private:
+    void _RelayTransaction(const uint256& txid, const uint256& wtxid)
+        EXCLUSIVE_LOCKS_REQUIRED(cs_main);
```

Comment:
9012512a57283bdcd0cc0562680d5de2f304af9f

Should include sync.h?

Reply:
Added